runtime/src/fips.rs

```diff
@@ -21,7 +22,16 @@ impl FipsModule {
     }
 
     pub fn shutdown(_env: &Drivers) -> CaliptraResult<MboxStatusE> {
-        cprintln!("[rt] FIPS shutdown");
-        Err(CaliptraError::RUNTIME_FIPS_UNIMPLEMENTED)
+        Self::zeroize();
```

Comment:
Note data vault entries and PCR registers are locked so we can't get rid of the state in those peripherals.